Sources/Attribution/AttributionFetcher.swift

```diff
@@ -19,6 +19,10 @@ import UIKit
 import WatchKit
 #endif
 
+#if os(iOS)
```

Comment:
Maybe
```suggestion
#if canImport(AdServices)
```

Reply:
oh cool, TIL. pushing most of the linking/library stuff off until we have more info, but this one seems worth doing for now :)